<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OpenFile">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>55</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>File is not selected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Convert">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Status">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>لم يتم البدء</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="B264">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>264</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="B265">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>265</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>314</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
